### ISSUE TITLE
Fix failing tests

### ISSUE THREAD

**Issue by gcroci2** [sdrogers/nplinker#96]
Not sure who assign to this. Even if the installation of nplinker package was successful, running the tests with pytest gave me errors in two tests:
```
FAILED tests/test_load_gnps.py::test_messy_strain_naming_lookup_raises[messy_alias0] - AttributeError: 'list' object has no attribute 'id'
FAILED tests/test_load_gnps.py::test_messy_strain_naming_lookup_raises[messy_alias1] - AttributeError: 'list' object has no attribute 'id'
```

**Comment by hechth** [comment]
That should be on my side - did you check also the branch of the open PR if stuff fails there as well?

**Comment by gcroci2** [comment]
> That should be on my side - did you check also the branch of the open PR if stuff fails there as well?

I run the tests from `dev` branch. To checkout to your forked repo's branch, should I just clone it or is there any cleaner way to checkout from nplinker repo to the PR coming from the forked one?

**Comment by hechth** [comment]
There is a nice way on github - if you open the PR in the tab and then there is a button that lets you check out a PR branch using the github cli

![image](https://user-images.githubusercontent.com/12066490/215728728-e1aa56a6-5d78-4bb4-9e35-a406b6fe5ec6.png)

**Comment by gcroci2** [comment]
Very handy, thanks :)

This is what I get from `gnps_downloader` branch:
```
FAILED tests/test_nplinker.py::test_load_data - assert 364 == 390
FAILED tests/test_nplinker.py::test_get_links - assert 58 == 101
```

**Comment by hechth** [comment]
@gcroci2 can you try running master by now since the PR was merged and confirm whether these differences are still there?

**Comment by gcroci2** [comment]
> @gcroci2 can you try running master by now since the PR was merged and confirm whether these differences are still there?

I've run tests in branch `dev` again (I noticed that you merged the PR #93 to this branch, not master). Now I get this:
```
ERROR tests/test_nplinker.py::test_load_data - AttributeError: 'NoneType' object has no attribute 'id'
ERROR tests/test_nplinker.py::test_get_links - AttributeError: 'NoneType' object has no attribute 'id'
```

**Comment by CunliangGeng** [comment]
```
ERROR tests/test_nplinker.py::test_load_data - AttributeError: 'NoneType' object has no attribute 'id'
ERROR tests/test_nplinker.py::test_get_links - AttributeError: 'NoneType' object has no attribute 'id'
```
The error is already solved in PR #102 and #105, which are under review. These PRs passed all local tests. So this issue should be solved.
